firmware/application/apps/ui_settings.cpp

```diff
@@ -259,11 +259,16 @@ SetUIView::SetUIView(NavigationView& nav) {
                   &toggle_converter,
                   &toggle_bias_tee,
                   &toggle_clock,
-                  &toggle_speaker,
+                  &toggle_mute,
                   &toggle_sd_card,
                   &button_save,
                   &button_cancel});
 
+    // Display "Disable speaker" option only if AK4951 Codec which has separate speaker/headphone control
```

Comment:
Optional: This will leave a gap in the little icon toggle widget on the settings page.
I guess that's ok, but it might be nice to have those automatically pack in using that StatusTray widget (see ui_navigation.cpp for how it's used).
I had added it here initially but took it out because it was more code.

Reply:
If it's Optional, I think I'll leave it and conserve the code space.  I thought it was probably OK to group the plain Status icons away from the Configuration icons, as long as you're OK with it.